### ISSUE TITLE
[usage question] How install ng2-dragula from fork?

### ISSUE THREAD

**Issue by Belouccio** [valor-software/ng2-dragula#953]
- [x] My issue title starts with `[usage question]`
- [x] I have read the [README](https://github.com/valor-software/ng2-dragula), especially the ['classic blunders' section](https://github.com/valor-software/ng2-dragula#classic-blunders)
- [x] I have looked at the [demos](https://valor-software.com/ng2-dragula/) to see if my use case has obvious examples.
- [x] I have searched the [`ng2-dragula` issues](https://github.com/valor-software/ng2-dragula/issues), including closed issues.
- [x] I have browsed [the issues labeled "future reference"](https://github.com/valor-software/ng2-dragula/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22future+reference%22) for problems that have been solved before.

**Can't install package via npm :(**

Hi. I need to use fork version of original 'dragula' (add 'mixed' direction to work it correctly with grid). I fork original 'dragula' and make changes, also I add fork version to dependencies of 'ng2-dragula' fork. But I can't install fork of 'ng2-dragula' via npm. I try like this
`npm i git+https://github.com/Belouccio/ng2-dragula.git ` but I got error 
`'Can't install git+https://github.com/Belouccio/ng2-dragula.git: Missing package name'`

I have also read and try to reproduce steps from this issue (https://github.com/valor-software/ng2-dragula/issues/748), but I think issue is with lerna monorepo.

Help me please)) I need 'mixed' direction, cause it work with grid much better then 'horizontal' or 'vertical' directions.

P.S. Thank you for your work and sorry for my bad English :)

**Comment by antgel** [comment]
@Belouccio Hey, I see you closed this the day after without providing a solution, @adiefrat and I are stuck in the same place, and #748 wasn't too clear for us either. How did you solve the issue? Thanks in advance.

**Comment by luisdpizarro** [comment]
@antgel I dont know if you have the same issue but I'll explain my situation.
Dragula had an issue I needed to fix, so I make a fork for the **dragula** npm package.
Then I make a fork for **ng2-dragula**, but this one didn't work because its at the modules/ subdirectory.

Solution:
I created a separate repo with the the content inside modules/ng2-dragula. I change de **dragula** version to my fork. And I find there were some issues with the Typescript version so I changed it to 2.8.4. And maybe you will need yarn so I add it it too at DevDependencies.
